java/src/jmri/NamedBeanBundle.properties

```diff
@@ -36,6 +36,9 @@ BeanNamePortal       = Portal
 EntryExit            = Entry Exit
 FastClock            = Fast Clock
 BeanName             = Named Bean
+# use the MakeLabel key to add a localized interpunction after the above (or any other) item
```

Comment:
The use of MakeLabel as a translatable string will only force this all to be redone later the first time the name of a bean needs to be different based on context. It will be significantly easier, in the long run, on both translators and developers, to create multiple strings in the properties files than to create formatters that only hold interpunction controls.

To quote [Apple's guidance](https://developer.apple.com/library/mac/documentation/MacOSX/Conceptual/BPInternational/InternationalizingYourCode/InternationalizingYourCode.html#//apple_ref/doc/uid/10000171i-CH4-SW1):

> Tip: Don’t overload keys or compose phrases from multiple keys. Some languages have gender articles, adjective endings, and completely different word order. Instead, add a separate key-value pair to the strings file for all unique phrases.
> For example, replace these key-value pairs:
>```
> /* Go to next page/chapter */
> "GoToNext" = "Go to next %@";
> "chapter" = "chapter";
> "page" = "page";
>```
> with separate key-value pairs for each phrase:
>```
> /* Go to next chapter */
> "GoToNextChapter" = "Go to next chapter";
>  
> /* Go to next page */
> "GoToNextPage" = "Go to next page";
>```
> Don’t put numbers in localizable strings because different regions can use different numbers.